deploy/backplane/mtsre/config.yaml

```diff
@@ -0,0 +1,4 @@
+deploymentMode: "SelectorSyncSet"
+selectorSyncSet:
+  matchLabels:
+      api.openshift.com/addon-managed-odh: "true"
```

Comment:
I think this is correct, the addon id is `managed-odh` and the others are prefixed `addon-`